app/views/install.scala.txt

```diff
@@ -45,8 +45,8 @@ sdkman_zshrc="${HOME}/.zshrc"
 
 sdkman_init_snippet=$( cat << EOF
 #THIS MUST BE AT THE END OF THE FILE FOR SDKMAN TO WORK!!!
-export SDKMAN_DIR="$SDKMAN_DIR"
-[[ -s "${SDKMAN_DIR}/bin/sdkman-init.sh" ]] && source "${SDKMAN_DIR}/bin/sdkman-init.sh"
+export SDKMAN_DIR="$HOME/.sdkman"
+[[ -s "$HOME/.sdkman/bin/sdkman-init.sh" ]] && source "$HOME/.sdkman/bin/sdkman-init.sh"
```

Comment:
However, this completely breaks how sdkman works when installing to alternate locations. The reason we use `SDKMAN_DIR` is so that sdkman can be installed in alternate locations to `HOME`. See the section [Installing to a custom location](https://sdkman.io/install) on the website.

I would be happy to find a solution for this, but we probably need to think it through a bit more. Please join our slack so we can chat about this.

Reply:
Almost forgot that one since Jan of this year already... OK, agreed. It's not the best solution here, but the first thing which came to my mind. :wink: Thanks for the invite, I'll come up with an alternative and join then.